yaptide/celery/utils/manage_tasks.py

```diff
@@ -110,12 +110,11 @@ def handle_cancellation_with_fetching(tasks: list[CeleryTaskModel]):
     if len(celery_ids_to_dump_data) > 0:
         for id in celery_ids_to_dump_data:
             result = AsyncResult(id)
-            result.backend.store_result(id, None, 'DUMP')
+            result.backend.store_result(id, {"dump": True}, state="RUNNING")
```

Comment:
how would that work if in the meantime task would go from running to completed state.Do you know if running that command will bring it back to running state ?

in this function (`handle_cancellation_with_fetching`) you fetch task state from the database, not from celery. Updates from celery to the DB are not so often, so it may happen as well that the celery task state at given moment of time is different that the task state in the DB.